builds/checkin/edgelet.yaml

```diff
@@ -124,65 +124,41 @@ jobs:
-#         displayName: Install Rust
-#       - script: |
-#           curl -L https://github.com/mozilla/grcov/releases/download/v0.5.1/grcov-linux-x86_64.tar.bz2 | tar jxf -
-#           curl -L https://raw.github.com/eriwen/lcov-to-cobertura-xml/8c55cd11f80a21e7e46f20f8c81fcde0bf11f5e5/lcov_cobertura/lcov_cobertura.py -o lcov_cobertura.py
-#         workingDirectory: edgelet
-#         displayName: Install code coverage tools
-#       - script: $CARGO build --verbose
-#         displayName: Build
-#         workingDirectory: edgelet
-#       - script: $CARGO test --verbose
-#         displayName: Test
-#         workingDirectory: edgelet
-#       - script: find ./target -type f ! -name '*.gc*' -delete
-#         displayName: Cleanup build and test artifacts
-#         workingDirectory: edgelet
-#       - script: |
-#           zip -1 target/ccov.zip `find ./target \( -name "*.gc*" \) -print`
-#           ./grcov target/ccov.zip -s . -t lcov --llvm --branch --ignore-not-existing --ignore-dir "/*" --ignore-dir "*docker-rs*" > target/lcov.info
-#           python lcov_cobertura.py target/lcov.info --output ./coverage.xml --demangle --base-dir $(Build.SourcesDirectory)/edgelet -e "${COVERAGE_EXCLUDES}"
-#         displayName: Assemble code coverage results
-#         workingDirectory: edgelet
-#         env:
-#           COVERAGE_EXCLUDES: $(coverage.excludes)
-#       - task: PublishCodeCoverageResults@1
-#         displayName: Publish code coverage results
-#         inputs:
-#           codeCoverageTool: cobertura
-#           summaryFileLocation: "edgelet/coverage.xml"
-#       - task: mspremier.BuildQualityChecks.QualityChecks-task.BuildQualityChecks@5
-#         displayName: "Check build quality"
-#         inputs:
-#           checkCoverage: true
-#           coverageFailOption: fixed
-#           coverageType: lines
-#           coverageThreshold: $(coverage.goal)
+################################################################################
+  - job: code_coverage
+################################################################################
+    displayName: Code Coverage
+    dependsOn: check_run_pipeline
+    condition: eq(dependencies.check_run_pipeline.outputs['check_files.RUN_PIPELINE'], 'true')
+    variables:
+      coverage.goal: 44
+    pool:
+      vmImage: "ubuntu-18.04"
+    steps:
+      - script: |
+          echo "##vso[task.setvariable variable=IOTEDGE_HOMEDIR;]/tmp"
+          echo "##vso[task.setvariable variable=CARGO;]${CARGO_HOME:-"$HOME/.cargo"}/bin/cargo"
+        displayName: Set env variables
+        workingDirectory: edgelet
+      - script: scripts/linux/generic-rust/install.sh --project-root "edgelet"
+        displayName: Install Rust
+      - script: |
+          $CARGO install cargo-tarpaulin
+        workingDirectory: edgelet
+        displayName: Install Cargo Tarpaulin
+      - script: |
+          $CARGO tarpaulin --out Xml --exclude-files "docker-rs*" --output-dir .
```

Comment:
let's create a variable for excluded files. I bet this list will grow soon enough.

Reply:
I think we only want to exclude code because it isn't a target for our tests, not because we don't think it's testable.